script.js

```diff
@@ -1,4 +1,452 @@
+  let currentNumber = currentNumberAFS(dice1, dice2);
+
+  if (userRound == user.length - 1 && accumulatedRound == 4) {
+    return AFSModeResult(dice1, dice2, currentNumber);
+  }
+  if (userRound == user.length - 1 && accumulatedRound < 4) {
+    userRound = 0;
+    accumulatedRound += 1;
+    winnerList = scorePointAndGenList(false);
+    return `${user[user.length - 1]} roll ${dice1} and ${dice2}.🎲🎲<br> The ${
+      currentNumber[1]
+    } combination number is ${
+      currentNumber[0]
+    }.<br><br>This is the end the Round ${accumulatedRound}.😎<br>Let's see the result until now.<br><br>${genUserNumberList()}<br>It will be hard for you${winnerList}.<br>The next round will use the smallest combination for the current winner.<br>Good Luck.😏😏`;
+  }
+
+  let result = `${
+    user[userRound]
+  } have rolled ${dice1} and ${dice2}.🎲🎲<br> The ${
+    currentNumber[1]
+  } combination number is ${currentNumber[0]}.<br>Next player ${
+    user[userRound + 1]
+  } please roll your dices!🤩🤩`;
+  userRound += 1;
+  return result;
+};
+
+var currentNumberAFS = function (dice1, dice2) {
+  let currentNumber = 0;
+  let largeOrSmall = "largest";
+  let output = [];
+  if (dice1 >= dice2 && accumulatedRound == 0) {
+    currentNumber = Number(String(dice1) + String(dice2));
+    playerNumber.push(currentNumber);
+  } else if (dice1 < dice2 && accumulatedRound == 0) {
+    currentNumber = Number(String(dice2) + String(dice1));
+    playerNumber.push(currentNumber);
+  } else if (dice1 >= dice2 && currentWinner.includes(userRound)) {
+    currentNumber = Number(String(dice2) + String(dice1));
+    playerNumber[userRound] += currentNumber;
+    largeOrSmall = "smallest";
+  } else if (dice1 < dice2 && currentWinner.includes(userRound)) {
+    currentNumber = Number(String(dice1) + String(dice2));
+    playerNumber[userRound] += currentNumber;
+    largeOrSmall = "smallest";
+  } else if (dice1 >= dice2) {
+    currentNumber = Number(String(dice1) + String(dice2));
+    playerNumber[userRound] += currentNumber;
+  } else if (dice1 < dice2) {
+    currentNumber = Number(String(dice2) + String(dice1));
+    playerNumber[userRound] += currentNumber;
+  }
+
+  output.push(currentNumber, largeOrSmall);
+  return output;
+};
+
+var AFSModeResult = function (dice1, dice2, currentNumber) {
+  let scorelist = scorePointAndGenList(true);
+  let result = `${dice1} and ${dice2} have rolled by ${
+    user[user.length - 1]
+  }.<br>The ${currentNumber[1]} combination number ${
+    currentNumber[0]
+  }have been used.🎲<br><br>5 Round of dices have been rolled!🎲🎲<br>Let's see the final result!😎<br><br>${genUserNumberList()}<br> Congrats${scorelist}🎉🎉🎉.You Win!<br>Now the score is in below:<br>${genUserList()}`;
+  endGame();
+  return result;
+};
+
+//reroll mode function
+var rerollModeRoll = function () {
+  let dice1 = genDice();
+  let dice2 = genDice();
+  playerDices = [dice1, dice2];
+  playerDices.sort(function (a, b) {
+    return b - a;
+  });
```

Comment:
Although unnecessary as there are better ways to do it, good job on reading up on array methods!